### ISSUE TITLE
Forwarded messages do not use E2EE

### ISSUE THREAD

**Issue by tnyeanderson** [tnyeanderson/sendmail-to-matrix#2]
Need to complete device verification before messages will be encrypted. This should ideally be an interactive and nearly automatic process.

https://matrix-nio.readthedocs.io/en/latest/examples.html#interactive-encryption-key-verification

**Comment by ekkehardd** [comment]
Really nice project, thanks! Just verified my device but it seems like it does not support E2EE by design, right? Even using from verified device.

**Comment by tnyeanderson** [comment]
You are correct, this project does not yet support E2EE in our usage of the Matrix API.

Which device did you verify, and how? There might be some way to add the device verification information to our calls if it is available, but I would have to check into it more.
